src/main/scala/beam/agentsim/infrastructure/ZonalParkingManager.scala

```diff
@@ -258,7 +262,14 @@ object ZonalParkingManager extends LazyLogging {
       case Some(result) =>
         result
       case None =>
-        val newStall = ParkingStall.lastResortStall(random)
+        val newStall =
+          ParkingStall
+            .defaultStall(destinationUTM)
```

Comment:
based on what you said, this could be the agent's home. however, if we just send them to the destination, then they may not learn that this was a bad route, because the remaining "score" of their day might lead them to believe they actually found parking. 

when the lastResortStall is produced, it really means, "ya, you're getting parking, but, you really shouldn't be."

Reply:
I know, that's why it's _expensive_. My point was that that should be enough, rather than _additionally_ have it be somewhere else, don't you think?